modules/administration/pages/tshoot-osadjabberd.adoc

```diff
@@ -84,6 +101,7 @@ spacewalk-service stop
 rm -rf /var/lib/jabberd/db/*
 spacewalk-service start
```

Comment:
We changed the used db since a while. Just remove everything and restart the service does nkt work anymore.
There is a command which create an empty sqlite db which must be used.

Reply:
`/usr/share/spacewalk/setup/jabberd/manage_database -s`

Reply:
Ok, this seems to be commented out now.